src/test/kotlin/uk/gov/dluhc/printapi/service/aed/AnonymousElectorDocumentSearchServiceIntegrationTest.kt

```diff
@@ -1,168 +1,188 @@
-package uk.gov.dluhc.printapi.database.repository
+package uk.gov.dluhc.printapi.service.aed
 
-import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
-import org.springframework.beans.factory.annotation.Autowired
-import org.springframework.data.domain.PageRequest
-import org.springframework.data.domain.Sort
 import uk.gov.dluhc.printapi.config.IntegrationTest
 import uk.gov.dluhc.printapi.dto.aed.AedSearchBy
+import uk.gov.dluhc.printapi.testsupport.assertj.assertions.Assertions.assertThat
 import uk.gov.dluhc.printapi.testsupport.testdata.aValidSourceReference
 import uk.gov.dluhc.printapi.testsupport.testdata.dto.aed.buildAnonymousSearchCriteriaDto
 import uk.gov.dluhc.printapi.testsupport.testdata.entity.buildAnonymousElectorDocument
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildElectoralRegistrationOfficeResponse
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildLocalAuthorityResponse
 import java.time.Instant
 import java.time.LocalDate
 
-@Deprecated("This is a temporary test, and will be removed when EIP1-5612 uses the specification builder in the search service")
```

Comment:
The diff is a bit hard to read here - basically what I've done is taken the temporary test `AnonymousElectorDocumentSummarySpecificationBuilderIntegrationTest` (which was always temporary), and deleted it.
And I've created a new test `AnonymousElectorDocumentSearchServiceIntegrationTest` which has the exact same test scenarios that the previous test had, but instead of hitting the repository class directly, it hits the service class and method 👍